### PR TITLE
v1.11.0

### PR DESCRIPTION
**Pull request by ozzieD**
v1.11.0 Jun 30, 2022
================
  * Enhancements
      * Add datetime and string types as valid arguments to dfs ``cutoff_time`` (#2147 ) 
      * Add ``PartOfDay`` transform primitive (#2128)
      * Add ``IsYearEnd``, ``IsYearStart`` transform primitives (#2124)
      * Add ``Feature.set_feature_names`` method to directly set output column names for multi-output features (#2142)
      * Include np.nan testing for ``DayOfYear`` and ``DaysInMonth`` primitives (#2146)
      * Allow dfs kwargs to be passed into ``get_valid_primitives`` (#2157)
  * Fixes
  * Changes
      * Improve serialization and deserialization to reduce storage of duplicate primitive information (#2136, #2127, #2144)
      * Sort core requirements and test requirements in setup cfg (#2152)
  * Documentation Changes
  * Testing Changes
      * Fix pandas warning and reduce dask .apply warnings (#2145)
      * Pin graphviz version used in windows tests (#2159)

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -2,8 +2,18 @@
...
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v1.11.0 June 30, 2022
+====================
```
Comment (thehomebrewnerd) [inline docs/source/release_notes.rst:16]: Underline needs to cover the entire text for docs to build

Comment (ozzieD) [reply docs/source/release_notes.rst]: changed

**Review by gsheni — DISMISSED** [review]

**Review by rwedge — COMMENTED** [review]
remove the fixes and documentation changes sections of the current release

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -2,8 +2,18 @@
...
+  ==============
+    * Enhancements
+    * Fixes
+    * Changes
+    * Documentation Changes
+    * Testing Changes
+
+.. Thanks to the following people for contributing to this release:
+
+v1.11.0 June 30, 2022
```
Comment (rwedge) [inline docs/source/release_notes.rst:15]: June should be Jun, this document always uses first 3 letters of the month

(other releases from this month should also be changed)

**Review by rwedge — COMMENTED** [review]
release 1.9.2 also has "June"

**Comment by ozzieD** [comment]
> release 1.9.2 also has "June"

also underline in v1.0.0 Oct 12, 2021 is one = too long

**Review by rwedge — APPROVED** [review]

**Pull request merged**